core/src/test/java/org/infinispan/topology/ClusterCacheStatusTest.java

```diff
@@ -0,0 +1,147 @@
+package org.infinispan.topology;
+
+import static java.util.Collections.singletonList;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+import static org.testng.AssertJUnit.assertEquals;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.infinispan.commons.hash.MurmurHash3;
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.distribution.TestAddress;
+import org.infinispan.distribution.ch.impl.DefaultConsistentHashFactory;
+import org.infinispan.manager.EmbeddedCacheManager;
+import org.infinispan.partitionhandling.AvailabilityMode;
+import org.infinispan.partitionhandling.impl.PreferAvailabilityStrategy;
+import org.infinispan.remoting.transport.Address;
+import org.infinispan.remoting.transport.Transport;
+import org.infinispan.statetransfer.RebalanceType;
+import org.infinispan.test.AbstractInfinispanTest;
+import org.infinispan.util.logging.events.EventLogManager;
+import org.infinispan.util.logging.events.impl.EventLogManagerImpl;
+import org.mockito.Mockito;
+import org.mockito.MockitoSession;
+import org.mockito.quality.Strictness;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class ClusterCacheStatusTest extends AbstractInfinispanTest {
+   private static final String CACHE_NAME = "test";
+   private static final CacheJoinInfo JOIN_INFO =
+      new CacheJoinInfo(new DefaultConsistentHashFactory(), MurmurHash3.getInstance(), 8, 2, 1000, false,
+                        CacheMode.DIST_SYNC, 1.0f, null, Optional.empty());
+   private static final Address A = new TestAddress(1, "A");
+   private static final Address B = new TestAddress(2, "B");
+   private static final Address C = new TestAddress(3, "C");
+
+   private ClusterCacheStatus status;
+   private ClusterTopologyManager topologyManager;
+   private MockitoSession mockitoSession;
+   private Transport transport;
+
+   @BeforeMethod(alwaysRun = true)
+   public void setup() {
+      mockitoSession = Mockito.mockitoSession().initMocks(this).strictness(Strictness.STRICT_STUBS).startMocking();
```

Comment:
Same thing with this one. Also I don't see any fields annotated with `@Mock` for `initMocks`?